rules/windows/credential_access_adidns_wildcard.toml

```diff
@@ -0,0 +1,82 @@
+[metadata]
+creation_date = "2024/03/26"
+integration = ["system", "windows"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2024/03/26"
+
+[rule]
+author = ["Elastic"]
+description = """
+Active Directory Integrated DNS (ADIDNS) is one of the core components of AD DS, leveraging AD's access control and
+replication to maintain domain consistency. It stores DNS zones as AD objects, a feature that, while robust, introduces
+some security issues, such as wildcard records, mainly because of the default permission (Any authenticated users) to
+create DNS-named records. Attackers can create wildcard records to redirect traffic that doesn't explicitly match
+records contained in the zone, becoming the Man-in-the-Middle and being able to abuse DNS similarly to LLMNR/NBNS spoofing.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-system.*", "logs-windows.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential ADIDNS Poisoning via Wildcard Record Creation"
+references = [
+    "https://www.netspi.com/blog/technical/network-penetration-testing/exploiting-adidns/",
+    "https://www.thehacker.recipes/a-d/movement/mitm-and-coerced-authentications/adidns-spoofing"
+]
+risk_score = 73
+rule_id = "8f242ffb-b191-4803-90ec-0f19942e17fd"
+setup = """## Setup
+
+The 'Audit Directory Service Changes' logging policy must be configured for (Success, Failure).
+Steps to implement the logging policy with Advanced Audit Configuration:
+
+```
+Computer Configuration >
+Policies >
+Windows Settings >
+Security Settings >
+Advanced Audit Policies Configuration >
+Audit Policies >
+DS Access >
+Audit Directory Service Changes (Success,Failure)
+```
+
+The above policy does not cover the target object by default (we still need it to be configured to generate events), so we need to set up an AuditRule using https://github.com/OTRF/Set-AuditRule.
+
+```
+Set-AuditRule -AdObjectPath 'AD:\\CN=MicrosoftDNS,DC=DomainDNSZones,DC=Domain,DC=com' -WellKnownSidType WorldSid -Rights CreateChild -InheritanceFlags Descendents -AttributeGUID e0fa1e8c-9b45-11d0-afdd-00c04fd930c9 -AuditFlags Success
+```
+"""
+severity = "high"
```

Comment:
Is this behavior that is always malicious? If not, might be better to bump down to medium and check telemetry before bumping up.